packages/orbit-components/src/primitives/MobileDialogPrimitive/__tests__/index.test.js

```diff
@@ -1,29 +1,37 @@
 // @flow
 import * as React from "react";
-import { mount } from "enzyme";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 import MobileDialog from "../index";
-import Button from "../../../Button";
 
-describe("MobileDialogPrimitive", () => {
-  const content = "My text link";
-  const tabIndex = "1";
-  const dataTest = "test";
+jest.mock("../../../utils/randomID", () => name => name);
 
-  const component = mount(
-    <MobileDialog tabIndex={tabIndex} dataTest={dataTest} content={content}>
-      <Button>Button</Button>
-    </MobileDialog>,
-  );
+let dialogs: HTMLElement;
 
-  const dialog = component.find("MobileDialog");
-  it("should contain a children", () => {
-    expect(dialog.children().exists()).toBe(true);
-  });
-  it("should have data-test", () => {
-    expect(dialog.prop("dataTest")).toBe(dataTest);
-  });
-  it("should have tabindex", () => {
-    expect(dialog.prop("tabIndex")).toBe(tabIndex);
+beforeAll(() => {
+  dialogs = document.createElement("div");
+  dialogs.setAttribute("id", "dialogs");
+  document.body?.appendChild(dialogs);
+});
+
+afterAll(() => {
+  document.body?.removeChild(dialogs);
+});
+
+describe("MobileDialogPrimitive", () => {
+  it("should have expected DOM output", async () => {
```

Comment:
seems like redundant `async`, there is no `await` inside and test passes without it

Reply:
You're right, it remained from something I previously tried.